boxsdk/object/item.py

```diff
@@ -358,6 +358,52 @@ def metadata(self, scope='global', template='properties'):
         """
         return Metadata(self._session, self, scope, template)
 
+    def get_watermark(self):
+        """
+        Return the watermark info for a Box file
+
+        :return:
+            Watermark object.
+        :rtype:
+            :class:`Watermark`
+        """
+        url = self.get_url('watermark')
+        box_response = self._session.get(url, expect_json_response=False)
```

Comment:
Why is this flag set?  It doesn't seem to make sense, since immediately below you call `.json()`